test/integration/bin/nested-path/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "bin": {
+    "c": "./dist/a/b/c.js",
```

Comment:
I think we can remove nested-path case as this is similar to multi-path, only difference is the path value. Rollup can make sure the files and folders are created correctly, we don't need to test it

Reply:
Got it!